sandbox-web/src/main/java/com/exadel/sandbox/team5/CommonController.java

```diff
@@ -0,0 +1,24 @@
+package com.exadel.sandbox.team5;
+
+import com.exadel.sandbox.team5.entity.BaseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+public interface CommonController<T extends BaseEntity> {
```

Comment:
What is the reason to create separate interface for controller?

Reply:
Application works without thit interface. It's an unnecessary component.